optimum/exporters/openvino/better_transformer_patch.py

```diff
@@ -0,0 +1,39 @@
+#  Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
would rename the file to `model_patcher.py` to keep the same convention with other `optimum` subpackages

Reply:
renamed